src/AstroBase.jl

```diff
@@ -481,8 +482,32 @@ function xy06(jd1, jd2)
         end
         ialast = ia - 1
     end
-
     sec2rad((xpr + (xyls[1] + xypl[1]) / 1e6)), sec2rad(ypr + (xyls[2] + xypl[2]) / 1e6)
 end
 
+"""
+    equations_of_origins(rnpb, s)
+
+Returns the equation of origins(radians) for given nutation-bias-precession matrix and the CIO locator.
+    mean_obliquity_of_ecliptic(jd1, jd2)
+
+Returns  obliquity of the ecliptic (radians) for a given Julian 2 part date (TT).
+
+# Example
+
+```jldoctest
+
+```
+"""
+function equation_of_origins(rnpb, s)
+    x = rnpb[1, 3]
+    ax = x / (1.0 + rnpb[3, 3])
+    xs = 1.0 - (ax * x)
```

Comment:
Actually, all parens beyond this point should be removed (except for the function call) 😂 https://en.wikipedia.org/wiki/Order_of_operations#Mnemonics 😜